lib/runtime/imports.go

```diff
@@ -248,7 +248,7 @@ func ext_storage_root(context unsafe.Pointer, resultPtr int32) {
 
 //export ext_storage_changes_root
 func ext_storage_changes_root(context unsafe.Pointer, a, b, c int32) int32 {
-	log.Trace("[ext_storage_changes_root] executing...")
+	log.Trace("[ext_storage_changes_root] executing...", "a", a, "b", b, "c", c)
```

Comment:
don't need the paramters for now

Reply:
since this is a Trace log, I thought it would be useful to add information about values passed into the function.

Reply:
okay that's fine